token/sdk/identity/provider.go

```diff
@@ -22,6 +23,14 @@ func NewKVSStorageProvider(sp view.ServiceProvider) *KVSStorageProvider {
 	return &KVSStorageProvider{sp: sp}
 }
 
-func (s *KVSStorageProvider) New(tmsID token.TMSID) (identity.Storage, error) {
+func (s *KVSStorageProvider) NewStorage(tmsID token.TMSID) (identity.Storage, error) {
```

Comment:
Maybe we could skip the error since it is always null. This way we will have a nil check less.

Reply:
In the current implementation, yes. But another might return an error 😓